app-laptop/msi-keyboard/msi-keyboard-1.0.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit udev
+
+DESCRIPTION="Control backlight of MSI laptop keyboards"
+HOMEPAGE="https://github.com/makkarpov/msi-keyboard"
+KEYWORDS="~amd64"
+LICENSE="GPL-3"
+SLOT="0"
+RDEPEND="dev-libs/hidapi"
+PATCHES=(
+		"${FILESDIR}"/${P}-cflags.patch
+)
+SRC_URI="https://github.com/makkarpov/msi-keyboard/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Your variables are all over the place, could you follow the default skel style? 
https://github.com/gentoo/gentoo/blob/master/skel.ebuild
```
# Copyright 2019 Gentoo Authors
# Distributed under the terms of the GNU General Public License v2

EAPI=7

DESCRIPTION=""
HOMEPAGE=""
SRC_URI=""

LICENSE=""
SLOT="0"
KEYWORDS="~amd64 ~x86"
IUSE=""

DEPEND=""
RDEPEND="${DEPEND}"
BDEPEND=""
```